draft-ietf-scitt-architecture.md

```diff
@@ -126,63 +126,47 @@ Within the SCITT Architecture, a producer is known as an Issuer, and a consumer
 
 # Introduction
 
-This document describes a scalable and flexible, decentralized architecture to enhance auditability and accountability across various existing and emerging supply chains.
-It achieves this goal by enforcing the following complementary security guarantees:
-
-1. Statements made by Issuers about supply chain Artifacts must be identifiable, authentic, and non-repudiable
-1. Such Statements must be registered on a secure Append-only Log, enabling provenance and history to be independently and consistently audited
-1. Issuers can efficiently prove to any other party the Registration of their Signed Statements; verifying this proof ensures that the Issuer is consistent and non-equivocal when producing Signed Statements
+This document describes the scalable, flexible, and decentralized SCITT architecture.
+Its goal is to enhance auditability and accountability across supply chains.
+
+In supply chains, there are producers and consumers of items.
```

Comment:
```suggestion
```

I would remove this line: it adds nothing and is arguably asking for trouble over definition of 'consumer' in particular. Plus what about the stages in the middle? Some people are fine calling these producers and consumers but many aren't.  The next line covers all of this without getting into trouble.

Reply:
If we trim line 132 above, the following 3 lines become orphaned. That may, or may not be a bad thing in the spirit of reduction to create focus, while still maintaining enough context for the goal